app/views/layouts/application.html.erb

```diff
@@ -39,6 +39,13 @@
       } %>
     <% end %>
 
+    <% if flash["alert_with_description"] %>
+      <%= render "govuk_publishing_components/components/error_alert", {
+        message: flash["alert_with_description"].fetch("title"),
+        description: sanitize(govspeak_to_html(flash["alert_with_description"].fetch("description")), tags: %w[a]),
```

Comment:
If I understand this right, we'll strip out all HTML that isn't links here? is there a particular reason we want to forbid any other elements? 

If so it'd probably useful to have this in a comment or commit message since it seems atypical to our usual handling of govspeak.

Reply:
Caught me! It was a hack I shouldn't have committed to get around styling issues caused by nested `<p>` tags. I fixed it in the components with https://github.com/alphagov/govuk_publishing_components/pull/517.